dag-sync.sh

```diff
@@ -30,6 +30,10 @@ have_dag=$(git log -p -1 "$new" --pretty=format: --name-only | grep "catalog/dag
 
 # Pull out the subject from the new commit
 subject=$(git log -1 --format='%s')
+# Swap the < & > characters for more complex unicode variants so they don't get
+# interpreted as delimiters by Slack
+subject=${subject//>/≻}
```

Comment:
Why not convert them to an HTML entity as described in the docs[^1]?

- Replace the less-than sign, < with `&lt;`
- Replace the greater-than sign, > with `&gt;`

[^1]: https://api.slack.com/reference/surfaces/formatting#escaping

Reply:
That is a _much_ better idea! I'll give that a shot 😁